packages/editors/vim/after/syntax/ocaml/styled.vim

```diff
@@ -0,0 +1,20 @@
+if polyglot#init#is_disabled(expand('<sfile>:p'), 'css', 'after/syntax/ocaml/styled.vim')
+  finish
+endif
+
+if exists('b:current_syntax')
+  let s:current_syntax = b:current_syntax
+  unlet b:current_syntax
+endif
+
+let b:styled_nested_syntax = 1
+syn include @CssSyntax syntax/css.vim
```

Comment:
Is there any way we can extend syntax/css.vim and add the interpolation? 
If it's not, does it sound a good idea to fork it? :S

Reply:
Yes, I think it does make sense to include interpolation to the CSS highlighting rules.
Added here: https://github.com/ahrefs/vim-styled-ppx/blob/main/syntax/styled-ppx-payload.vim#L524-L525